### PR TITLE
Fixing error with names in upgrade test

### PR DESCRIPTION
**Pull request by manasagowri**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml [hunk 1]
```diff
@@ -178,7 +178,7 @@ tests:
       destroy-cluster: false
       abort-on-fail: true
   - test:
-      name: Upgrade along with I/O's
+      name: Upgrade along with IOs
```
Comment (sunilangadi2) [inline suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml:181]: @manasagowri Sorry i don't know that using test name it's going to create log file
```

> Traceback (most recent call last):
> 
>   File "run.py", line 1026, in <module>
> 
>     rc = run(args)
> 
>   File "run.py", line 741, in run
> 
>     tc["log-link"] = configure_logger(unique_test_name, run_dir)
> 
>   File "/home/jenkins-build/workspace/rhceph-cron-pipeline-test-executor/utility/utils.py", line 458, in configure_logger
> 
>     _handler = logging.FileHandler(test_logfile)
> 
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1032, in __init__
> 
>     StreamHandler.__init__(self, self._open())
> 
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1061, in _open
> 
>     return open(self.baseFilename, self.mode, encoding=self.encoding)
> 
> FileNotFoundError: [Errno 2] No such file or directory: "/ceph/cephci-jenkins/cephci-run-DUJTEE/Upgrade_along_with_I/O's_0.log

 ```

yes due to this PR https://github.com/red-hat-storage/cephci/commit/6b6f850acab00aed5346cbf1280b8f17425327b1
Tests getting failed for that / in IO's, But I have updated like this name for other upgrade suites 
Please refer to the attached PR here and make all other suites like this name
Thanks for the quick fix.

**Review by sunilangadi2 — CHANGES_REQUESTED** [review]

File: suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml (hunk 1 above)
Comment (manasagowri) [reply suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml:181]: Thanks Sunil. I made the changes in all relevant test suites.

**Review by sunilangadi2 — APPROVED** [review]
LGTM

File: suites/pacific/cephadm/tier-0_cephadm.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ tests:
       desc: installation of ceph pre-requisites
       module: install_prereq.py
       abort-on-fail: true
+      config:
+        is_production: true
```
Comment (sunilangadi2) [inline suites/pacific/cephadm/tier-0_cephadm.yaml:23]: @manasagowri Is this suite runs against live repo, Can you please let me know why **is_production** set for this suite?

Comment (manasagowri) [reply suites/pacific/cephadm/tier-0_cephadm.yaml]: Removed it

**Review by psathyan — APPROVED** [review]

**Pull request merged**